e2e/__tests__/__snapshots__/jest.config.js.test.ts.snap

```diff
@@ -1,36 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
why are these deleted?

Reply:
Not intended, I had to recreate them several times and I've omitted to add them in the last commit.